profiling/src/lib.rs

```diff
@@ -692,6 +733,41 @@ extern "C" fn mshutdown(r#type: c_int, module_number: c_int) -> ZendResult {
         profiler.stop();
     }
 
+    #[cfg(feature = "allocation_profiling")]
+    {
```

Comment:
I think we should just skip the whole block `if zend::is_zend_mm()`. Don't even try (maybe another extension reset them, then that's also fine and we have nothing to do here), and don't emit two info!() warnings.